chronograf.go

```diff
@@ -380,7 +380,7 @@ type Dashboard struct {
 
 // DashboardCell holds visual and query information for a cell
 type DashboardCell struct {
-	ID      string           `json:"-"`
+	ID      string           `json:"i"`
```

Comment:
Agree w/ @goller here, we should try to use `links.self`. `react-grid-layout` is _very_ finicky about that `i` param though... you might need to go out of your way to coerce that into something it likes.